themes/default/content/docs/guides/crosswalk/aws/vpc.md

```diff
@@ -189,7 +189,7 @@ import pulumi_awsx as awsx
 
 vpc = awsx.ec2.Vpc("custom")
 
-pulumi.export("vpcId", vpc.vpc_id)
+pulumi.export("vpc_id", vpc.vpc_id)
```

Comment:
Sorry to be difficult, but we'd rather you left the exported Output name the same (i.e., leave it as `vpcId`) and just change the resource and property as needed. Can you revert the changes to the exports?